frontend/.env.local

```
@@ -125,4 +132,4 @@ I18N_LNG=en
 NEXT_PUBLIC_DEVICE_INSTALL_COMMAND=curl -fsSL https://raw.githubusercontent.com/wecode-ai/Wegent/main/install-wework.sh | bash
 
 # NEXT_PUBLIC_DEVICE_GUIDE_URL: URL to device setup guide documentation
-NEXT_PUBLIC_DEVICE_GUIDE_URL=https://github.com/wecode-ai/Wegent
+NEXT_PUBLIC_DEVICE_GUIDE_URL=https://github.com/wecode-ai/Wegent
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)